src/librustc_mir/borrow_check.rs

```diff
@@ -1169,8 +1169,70 @@ impl<'c, 'b, 'a: 'b+'c, 'gcx, 'tcx: 'a> MirBorrowckCtxt<'c, 'b, 'a, 'gcx, 'tcx>
         err.emit();
     }
 
+    /// Finds the span of arguments of aclosure (within `maybe_closure_span`) and its usage of
+    /// the local assigned at `location`.
+    fn find_closure_span(
```

Comment:
It'd be nice to add some comments explaining _what_ is being done in the code, but this method is short enough that it is not _necessary_.